lib/default-profile.js

```diff
@@ -37,7 +37,7 @@ import {formatTravellers} from '../format/travellers.js';
 import {formatLoyaltyCard} from '../format/loyalty-cards.js';
 import {formatTransfers} from '../format/transfers.js';
 
-const DEBUG = (/(^|,)hafas-client(,|$)/).test(process.env.DEBUG || '');
+const DEBUG = (/(^|,)hafas-client(,|$)/).test((typeof process !== 'undefined') ? (process.env.DEBUG || '') : '');
```

Comment:
You can use [the optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

```suggestion
const DEBUG = (/(^|,)hafas-client(,|$)/).test(process?.env?.DEBUG ?? '');
```

Reply:
I have tried your exact change, but it did still throw an error, that `process` is undefined. My guess was that undefined chaining does not work on the root variable. However I might be wrong and if you tell me that it should work I might just try it again.

Reply:
And it actually says exactly that further down in the docs that you have linked.

Reply:
Huh, I didn't know this!

Another completely different approach to solve this – which is quite widespread in the JS frontend tooling landscape, but *requires* the consumers of `db-vendo-client` to configure/use it – is to replace all `process.env.*` expressions with their environment variables' values at build time. (see [esbuild's "Platform" docs section](https://esbuild.github.io/api/#platform) & [webpack's `EnvironmentPlugin`](https://webpack.js.org/plugins/environment-plugin/)).

Reply:
Yes of course, but as process.env only has one occurrence in the project right now, I have decided that this little ugly solution is worth it, so no one has to configure webpack.